lightcurve/views.py

```diff
@@ -175,3 +177,26 @@ def all_lightcurve(request):
 
     return render_to_response('lightcurve_list.html', {"contexts": context_list, "request_all": True},
             context_instance=RequestContext(request))
+
+
+def download(request, lightcurve_id):
+    file_type = request.GET.get('file_type')
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment; filename="LightCurve' + lightcurve_id + '.' + file_type + '"'
+
+    lc = LightCurve.objects.get(id=lightcurve_id)
+    analyses = ImageAnalysis.objects.filter(useruploadedimage__lightcurve=lc) \
+                                    .exclude(status=ImageAnalysis.ASTROMETRY_PENDING)
+
+    if file_type == "csv":
```

Comment:
Let's use single quotes, here and below

Reply:
Done.
Had to change `result["mag_instrumental"]` to `result.get('mag_instrumental', None)` though. Double quote didn't work for `result[]` because the data's in unicode. `.get()` don't have the same problem.